docs/post-mortems/0002-terminating-nodes.md

```diff
@@ -0,0 +1,95 @@
+## Summary
+
+Most of the pods running in the development environment of team Booking was down for around 1h 30min. 
+
+The cause is the same as an earlier
+[incident](https://docs.google.com/document/d/1_NRW32xvu7YgEakl1JnLf2nDRbOPe8exrzgnwPAAj30/edit#heading=h.mwz2ai1qrpl4).
+
+## Timeline
+
+* 2022-01-07 12:39 - 15:30
+  * 12:39 A user from team Booking notifies us on Slack that their development environment is down. We immediately start
+investigating.
+
+  * We identify the cause to be missing resource quotas. This has been fixed in with a
+    [resource quota patch](https://github.com/oslokommune/okctl-upgrade/blob/main/gists/resource-quota-patch/resource-quota-patch.sh),
+    but it has not been run in this cluster, so we run it.
+
+  * We identify that the patch for
+    [persistent logs for Loki](https://github.com/oslokommune/okctl-upgrade/tree/main/gists/loki-persistent-storage) has not
+    been run. This complicates putting resource quotas on the Loki pods, because logs will disappear as soon as the quotas
+    are applied, because the pod will restart.
+
+  * In AWS console, we set Auto Scaling Groups min and max capacity from 1 to 0, and then back again. At the time, we had an
+  idea it would fix issues with the node. In hindsight, this was probably not necessary.
+
+  * We run the resource quota patch to set resource quotas on most pods except Loki.
+
+  * We create a backup of Loki logs in development, and test that it's possible to restore. We then apply the persistent
+    storage patch for Loki, and restore the logs. We then run the resource quota patch again, this time enabling it for Loki also.
+
+  * We then run the resource quota patch, without Loki, in production. As it's getting late, we delay the rest of the fix to
```

Comment:
Do you have any timestamps for the other actions? Or an end time for the support, so we can see how much time was spent on the incident?

Reply:
Done!